Dangerfile

```diff
@@ -3,7 +3,7 @@ PR_SIZE = {
   RECOMMENDED_MAXIMUM: 250,
   ABSOLUTE_MAXIMUM:    1000,
 }
-EXCLUSIONS = ['Gemfile.lock', '.json', 'spec/fixtures/', '.txt', 'spec/support/vcr_cassettes/', 'app/swagger']
+EXCLUSIONS = ['Gemfile.lock', '.json', 'spec/fixtures/', '.txt', 'spec/support/vcr_cassettes/', 'app/swagger', 'modules/mobile/docs/index.html']
```

Comment:
is `index.html` the only file that gets generated in this code?  or could there be other files generated in `/docs`?

Reply:
Good callout. Right now it's only index.html, but I can imagine the possibility of others in this dir. We should exclude the entire docs directory:
```suggestion
EXCLUSIONS = ['Gemfile.lock', '.json', 'spec/fixtures/', '.txt', 'spec/support/vcr_cassettes/', 'app/swagger', 'modules/mobile/docs/']
```

Reply:
Looks good, merged it